### ISSUE TITLE
-exInputFile doesn't work

### ISSUE THREAD

**Issue by casella** [OpenModelica/OpenModelica#9488]
I am trying to figure out how CSV-input files actually work to document them, see #9472. Apparently, you can use either [-csvInput](https://www.openmodelica.org/doc/OpenModelicaUsersGuide/latest/simulationflags.html#simflag-csvinput) or [-exInputFile](https://www.openmodelica.org/doc/OpenModelicaUsersGuide/latest/simulationflags.html#simflag-exinputfile) to get data from an input file to top-level inputs of a model.

I am using this MWE
```modelica
model TestTopInputs
  Modelica.Blocks.Interfaces.RealInput u1;
  Modelica.Blocks.Interfaces.RealInput u2;
  Modelica.Blocks.Interfaces.RealInput u3;
  Real y = 2*u1;
  Real z = 4*u2;
  Real w = u1*u2*u3;
  annotation(
    uses(Modelica(version = "4.0.0")),
    experiment(StartTime = 0, StopTime = 5, Tolerance = 1e-06, Interval = 0.01));
end TestTopInputs;
```
I am simulating this MWE with `-exInputFile=abc.csv`, where `abc.csv` is a file that needs to be in the same directory as TestTopInput.mo (see also #9482). I am putting the input data u1, u2, and u3 in reverse order in the csv file, to make sure that the column are actually associated to the corresponding variable, and not just ordered lexicographically or something.

I tried this `abc.csv` file
```
time, u3, u2, u1
0, 0, 0, 0
1, 0, 0, 0
2, 0, 0, 1
3, 0, 2, 1
4, 3, 1, 1
5, 1, 1, 1
```
and this abc.csv file
```
time, u3, u2, u1
0.0, 0.0, 0.0, 0.0
1.0, 0.0, 0.0, 0.0
2.0, 0.0, 0.0, 1.0
3.0, 0.0, 2.0, 1.0
4.0, 3.0, 1.0, 1.0
5.0, 1.0, 1.0, 1.0
```
in both cases, all variables of the model were zero. 

I also tried to use other separators instead of the comma, but I always get the same bogus result.

If I understand correctly, the function that manages -exInputFile is [external_input.c](https://github.com/OpenModelica/OpenModelica/blob/61bb5d5d4e425e9c58bbfc2ade95870af81e6c5b/OMCompiler/SimulationRuntime/c/simulation/solver/external_input.c#L160). Maybe there is some obvious mistake there that can be fixed quickly. Otherwise, I would recommend to remove this simulation option, because it could be really confusing.

**Comment by mahge** [comment]
This should be fixed with #9518. The option is removed. 

For the record the only differences between the options `-exInputFile` and `-csvInput` were:

  - `exInputFile` accepted space separated CSV files **only**.
  - `exInputFile` always looked for a default file called `externalInput.csv` if no file was explicitly specified.

**Comment by casella** [comment]
OK. In fact, this was rather confusing. 

For the record one can get now the same feature with `-csvInput=externalInput.csv`, except that it now works 😄
